src/ontology/mondo-edit.obo

```diff
@@ -444446,16 +444418,29 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 [Term]
 id: MONDO:0020772
 name: epilepsy, juvenile absence, susceptibility to, 1
+subset: gard_rare {source="GARD:2162"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: ordo_disease {source="Orphanet:1941"}
+subset: orphanet_rare {source="Orphanet:1941"}
+subset: rare
 synonym: "EJA1" EXACT ABBREVIATION [OMIM:607631]
 synonym: "EPILEPSY, JUVENILE ABSENCE, SUSCEPTIBILITY TO, 1" EXACT [OMIM:607631]
+synonym: "epilepsy, juvenile absence, susceptibility to, type 1" EXACT [MONDORULE:1, OMIM:607631]
 synonym: "JAE1" EXACT ABBREVIATION [OMIM:607631]
+synonym: "susceptibility to juvenile absence epilepsy 1" EXACT [OMIM:607631]
 xref: DOID:0111324 {source="MONDO:equivalentTo"}
 xref: OMIM:607631 {source="MONDO:equivalentTo"}
 xref: Orphanet:1941 {source="OMIM:607631"}
-xref: UMLS:C2750892 {source="OMIM:607631"}
+xref: UMLS:C2750892 {source="MONDO:notFoundInDiseaseSubset", source="OMIM:607631"}
 is_a: MONDO:0020573 {source="OMIM:607631"} ! inherited disease susceptibility
-relationship: excluded_subClassOf MONDO:0011876 {source="OMIM:607631", source="https://orcid.org/0000-0001-5208-3432"} ! juvenile absence epilepsy
-relationship: predisposes_towards MONDO:0011876 {source="OMIM:607631"} ! juvenile absence epilepsy
+is_a: MONDO:1030001 ! epilepsy, juvenile absence, susceptibility to
```

Comment:
This needs a source (either OMIM or your ORCID)

Reply:
fixed with https://github.com/monarch-initiative/mondo/pull/7320/commits/09d53e7a3e385c5f8a87a0648bfd5870a91a0130